MainApp/lib/app/routes/api.js

```diff
@@ -285,7 +288,27 @@ export default function (router) {
         });
     });
 
-    router.get("/exportAdminLogs", function (req, res) {
+    router.post("/addToAdminLog", function (req, res) {
+        let adminLog = new AdminLog();
+        if(req.body.fullname || req.body.username) {
+            adminLog.author = req.body.fullname? req.body.fullname: req.body.username;
```

Comment:
@yousuf031991  a small question. Any special reason for using full name as author? There is a very thin chance that two admins can have same full name. (Note: I was just going through the code to understand it not review it.)